web/components/PetPostModal/PetPostModal.tsx

```diff
@@ -125,11 +191,35 @@ const FosterQuestionnaire = ({
         <ModalCloseButton display={["none", "block"]} mt={6} mr={6} />
         <Flex display={["none", "flex"]} flexDir="column" overflowY="scroll">
           <Flex paddingX={10} flexDir="column" gap={4}>
-            {data[fosterType].map((question) => {
+            {data[fosterType].map(({ key, title }) => {
               return (
-                <Flex key={question.key} flexDir="column" gap={2}>
-                  <Text>{question.title}</Text>
-                  <Input />
+                <Flex key={key} flexDir="column" gap={2}>
+                  <FormControl
+                    isRequired
+                    isInvalid={
+                      fosterQuestionResponses[
+                        key as keyof typeof QuestionKeys
+                      ] === ""
+                    }
```

Comment:
Let's remove this - let's only validate the responses when a user tries to submit the form.